services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -200,6 +202,13 @@ type SideBarListItemProps = {
 export const SideBarListItem: React.FC<{ children: BatchAction }> = ({ children: action }: SideBarListItemProps) => {
     const { environmentLocks, appLocks } = useAllLocks();
     const actionDetails = getActionDetails(action, appLocks, environmentLocks);
+    var displayVersion = '';
```

Comment:
Also I'd rather have this as a `const` if possible

Reply:
This can be omitted if later on you specify `const displayVersion = useRelease(actionDetails.application ?? '', actionDetails.version ?? 0).displayVersion | '';`

Reply:
Since useRelease() is not exported I am unable to make the change specified